From: Michael Froman <mfroman@mozilla.com>
Date: Mon, 29 Dec 2025 15:42:34 -0600
Subject: (tmp-cherry-pick) Revert "First stab at adding clang-tidy bot."
 (4774c16b70)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 02acc3ded1ae87086fc2964b84d1ab9a8abe7433.

Reason for revert: let's try to not reuse chromium's bot for this

Original change's description:
> First stab at adding clang-tidy bot.
>
> Change-Id: Idfd9a6d8cdc1c93c7b7d59233dd609e16de7df1a
> Bug: webrtc:452866579
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/418040
> Reviewed-by: Björn Terelius <terelius@webrtc.org>
> Commit-Queue: Jeremy Leconte <jleconte@google.com>
> Cr-Commit-Position: refs/heads/main@{#45970}

Bug: webrtc:452866579
Change-Id: Ia781af65d62650b0e330953848403ea393a49ada
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/418960
Bot-Commit: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Commit-Queue: Jeremy Leconte <jleconte@google.com>
Reviewed-by: Björn Terelius <terelius@webrtc.org>
Cr-Commit-Position: refs/heads/main@{#45987}
---
 infra/config/builders.star                   | 15 ---------
 infra/config/config.star                     |  4 ---
 infra/config/generated/luci/commit-queue.cfg | 33 --------------------
 3 files changed, 52 deletions(-)

diff --git a/infra/config/builders.star b/infra/config/builders.star
index 94ba64a05a..fd62c53928 100644
--- a/infra/config/builders.star
+++ b/infra/config/builders.star
@@ -504,18 +504,3 @@ cron_builder(
     },
     schedule = "*/10 * * * *",  # Every 10 minutes.
 )
-
-luci.cq_tryjob_verifier(
-    cq_group = "cq",
-    builder = "chromium:try/tricium-clang-tidy",
-    owner_whitelist = ["project-webrtc-tryjob-access"],
-    experiment_percentage = 0,
-    disable_reuse = True,
-    mode_allowlist = [cq.MODE_NEW_PATCHSET_RUN],
-    location_filters = [
-        cq.location_filter(path_regexp = r".+\.h"),
-        cq.location_filter(path_regexp = r".+\.c"),
-        cq.location_filter(path_regexp = r".+\.cc"),
-        cq.location_filter(path_regexp = r".+\.cpp"),
-    ],
-)
diff --git a/infra/config/config.star b/infra/config/config.star
index ea751e2ad6..592e2d4521 100755
--- a/infra/config/config.star
+++ b/infra/config/config.star
@@ -80,10 +80,6 @@ luci.project(
         ),
         acl.entry(acl.LOGDOG_WRITER, groups = ["luci-logdog-chromium-writers"]),
         acl.entry(acl.SCHEDULER_OWNER, groups = ["project-webrtc-admins"]),
-        acl.entry(
-            roles = acl.CQ_NEW_PATCHSET_RUN_TRIGGERER,
-            groups = "project-webrtc-tryjob-access",
-        ),
     ],
     bindings = [
         luci.binding(
diff --git a/infra/config/generated/luci/commit-queue.cfg b/infra/config/generated/luci/commit-queue.cfg
index 9faac94337..b7c3a0c960 100644
--- a/infra/config/generated/luci/commit-queue.cfg
+++ b/infra/config/generated/luci/commit-queue.cfg
@@ -24,43 +24,12 @@ config_groups {
     gerrit_cq_ability {
       committer_list: "project-webrtc-submit-access"
       dry_run_access_list: "project-webrtc-tryjob-access"
-      new_patchset_run_access_list: "project-webrtc-tryjob-access"
       allow_owner_if_submittable: DRY_RUN
     }
     tree_status {
       url: "https://webrtc-status.appspot.com"
     }
     tryjob {
-      builders {
-        name: "chromium/try/tricium-clang-tidy"
-        disable_reuse: true
-        location_filters {
-          gerrit_host_regexp: ".*"
-          gerrit_project_regexp: ".*"
-          gerrit_ref_regexp: ".*"
-          path_regexp: ".+\\.h"
-        }
-        location_filters {
-          gerrit_host_regexp: ".*"
-          gerrit_project_regexp: ".*"
-          gerrit_ref_regexp: ".*"
-          path_regexp: ".+\\.c"
-        }
-        location_filters {
-          gerrit_host_regexp: ".*"
-          gerrit_project_regexp: ".*"
-          gerrit_ref_regexp: ".*"
-          path_regexp: ".+\\.cc"
-        }
-        location_filters {
-          gerrit_host_regexp: ".*"
-          gerrit_project_regexp: ".*"
-          gerrit_ref_regexp: ".*"
-          path_regexp: ".+\\.cpp"
-        }
-        owner_whitelist_group: "project-webrtc-tryjob-access"
-        mode_allowlist: "NEW_PATCHSET_RUN"
-      }
       builders {
         name: "webrtc-internal/g3.webrtc-internal.try/internal_compile"
         includable_only: true
@@ -256,7 +225,6 @@ config_groups {
     gerrit_cq_ability {
       committer_list: "project-webrtc-submit-access"
       dry_run_access_list: "project-webrtc-tryjob-access"
-      new_patchset_run_access_list: "project-webrtc-tryjob-access"
     }
     tryjob {
       builders {
@@ -425,7 +393,6 @@ config_groups {
     gerrit_cq_ability {
       committer_list: "project-webrtc-admins"
       dry_run_access_list: "project-webrtc-tryjob-access"
-      new_patchset_run_access_list: "project-webrtc-tryjob-access"
     }
     tryjob {
       builders {
